holodex/dhen-data/org.json

```diff
@@ -153,29 +144,23 @@
                     "name": "trainee",
                     "pluralName": "trainees",
                     "relationshipType": "relationshipTypes/Trainer",
-                    "linkType": "linkTypes/trainee-of",
-                    "label": "role 2 label"
+                    "labels": " a trainee of {relation}"
                 }
             ],
             "links": [
                 {
                     "@type": "LinkType",
-                    "id": "linkTypes/trainer-for",
+                    "id": "linkTypes/trains",
                     "relationshipType": "relationshipTypes/Trainer",
                     "source": "roleTypes/trainer",
                     "target": "roleTypes/trainee",
-                    "label": "link 1 label"
-                },
-                {
-                    "@type": "LinkType",
-                    "id": "linkTypes/trainee-of",
-                    "relationshipType": "relationshipTypes/Trainer",
-                    "source": "roleTypes/trainee",
-                    "target": "roleTypes/trainer",
-                    "label": "link 2 label"
+                    "labels": "{source} trains {target}",
+                    "queryLabels": "People who train {target}, People trained by {source}, People whom {source} trains, People who train others for {context}"
                 }
             ]
         },
+        /* Question: Sites may be the context for relationships?*/
+
```

Comment:
as I note above, the sites might be  'contexts' for relationships. E.g a [grower] -> [grows] -> [grown] [in the context/for] [organisation]. Or it might be as you have it: [grower] -> [grows-for] -> [organisation].

cc @ahdinosaur